tests/unit/test_charm.py

```diff
@@ -18,11 +18,15 @@ def setUp(self):
         self.addCleanup(self.harness.cleanup)
         self.harness.begin()
 
+    @patch(
+        "test_charm.MagmaAccessGatewayOperatorCharm._is_magmad_enabled", new_callable=PropertyMock
```

Comment:
Ideally we would not mock our own private class but the return of the `systemctl` command

Reply:
Fixed